src/Twigc/DefaultCommand.php

```diff
@@ -399,6 +413,10 @@ public function doRender(InputInterface $input, OutputInterface $output) {
 				'autoescape'       => $escape,
 			]);
 
+			$twig->getExtension('core')->setEscaper('json', function($twigEnv, $string, $charset) {
```

Comment:
Can you format this as follows, so that the constants are fully qualified and the lines stay under 80 cols?

```php
$twig->getExtension('core')->setEscaper(
	'json',
	function($twigEnv, $string, $charset) {
		return json_encode(
			$string,
			\JSON_UNESCAPED_SLASHES | \JSON_UNESCAPED_UNICODE
		);
	}
);
```